hack/release/pkg/updatecapimate/updatecapimate.go

```diff
@@ -0,0 +1,38 @@
+package updatecapimate
+
+import (
+	"fmt"
+	"os"
+	"path/filepath"
+	"strings"
+
+	"github.com/mesosphere/kommander-applications/hack/release/pkg/constants"
+)
+
+func UpdateCAPIMateVersion(kommanderApplicationsRepo, chartVersion string) error {
+	pathToChart := filepath.Join(constants.KommanderAppPath, "*/*/cm.yaml")
+	matches, err := filepath.Glob(filepath.Join(kommanderApplicationsRepo, pathToChart))
+	if err != nil {
+		return err
+	}
+	if len(matches) == 0 {
+		return fmt.Errorf("no matches found for path %s (verify the kommander-applications repo path is correct)", pathToChart)
+	}
+	if len(matches) > 1 {
+		return fmt.Errorf("found > 1 match for path %s (there should only be one match)", pathToChart)
+	}
```

Comment:
You can combine it with `len(matches) != 1` to have a fewer lines to cover with tests.